frontend/src/components/organisms/ResetPasswordForm.tsx

```diff
@@ -54,7 +90,7 @@ const ResetPassword = () => {
         />
       </div>
       <div>
-        <Button>Reset Password</Button>
+        <Button type="submit">Reset Password</Button>
```

Comment:
the`useUpdatePassword` hook from react-firebase-hooks should provide some isLoading Prop that we can pass into the loading prop of this button. So when the user clicks the button, they get some feedback that something is happening. Here's an excerpt from LoginForm

```js
  /** Firebase hooks */
  const [
    signInWithEmailAndPassword,
    signedInUser,
    signInLoading,
    signInErrors,
  ] = useSignInWithEmailAndPassword(auth);

...

<Button
      loading={signInLoading}
      disabled={signInLoading}
      type="submit">
            Log in
  </Button>
```